docs/modules/ROOT/pages/explanations/decisions/postgresql.adoc

```diff
@@ -0,0 +1,84 @@
+= PostgreSQL Operators
+
+== Problem
+
+Creating our own operator to deploy and manage PostgreSQL proved to be a gargantuan effort. In order to keep in line with the newly specified https://kb.vshn.ch/app-catalog/reference/framework-requirements.html#_for_service_maintainer["more buy than make"] approach, we're evaluating the viability of using existing operators for the use of converged services.
```

Comment:
Maybe state a few more issues we've seen, besides the effort:
* Why build on our own when there are already plenty of viable alternatives around?
* Unpredictable amount of maintenance needed to keep our own solution up-to-date and maintained
* What do we gain when building our own vs. using an existing solution and integrating it into our systems